packages/react-app/src/pages/participants.tsx

```diff
@@ -28,6 +28,7 @@ const ParticipantsManagerPage = () => {
   const [giveaway, setGiveaway] = useState<Giveaway>();
   const [participants, setParticipants] = useState(participantsData);
   const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState<string>('no action');
```

Comment:
I don't think the actual state is important here, if the admin approves a participant we should prevent to reject that same participant while the operation is still in progress, right?